src/panel/poi/blocks/Covid19.jsx

```diff
@@ -43,7 +43,7 @@ const getContent = (poi, { status, opening_hours, note, contribute_url }) => {
       <div className="covid19-status covid19-status--open">{covidStrings.statusOpen}</div>
       {schedule && <div className="covid19-timeTableContainer">
         <i className="icon-icon_clock" />
-        <TimeTable title={<OpeningHour poi={poi} />} schedule={schedule} />
+        <TimeTable title={<OpeningHour openingHours={opening_hours} />} schedule={schedule} />
```

Comment:
This should allow some simplification, by not requiring `poi` to be passed at all in this function and even this component. Much cleaner indeed!

Reply:
Done